pkg/operator/status/condition.go

```diff
@@ -95,6 +95,9 @@ func unionMessage(conditions []operatorv1.OperatorCondition) string {
 
 func unionReason(conditions []operatorv1.OperatorCondition) string {
 	if len(conditions) == 1 {
+		if len(conditions[0].Reason) != 0 {
+			return fmt.Sprintf("%s: %s", conditions[0].Type, conditions[0].Reason)
+		}
 		return conditions[0].Type
```

Comment:
`return conditions[0].Type+conditions[0].Reason` I htink

Reply:
Thought we want it in the same format as https://github.com/openshift/library-go/blob/master/pkg/operator/status/condition.go#L90
Not sure if just joining Type and Reason without any additional punctuation right.